src/Layout/ComputationMenu/YearSelect.tsx

```diff
@@ -3,6 +3,7 @@ import * as React from "react";
 export interface YearSelectProps {
     electionYears: string[];
     year: string;
+    tooltip?: React.ReactNode;
```

Comment:
This seems quite vague/"wide" in type terms, is there a reason it's a ReactNode rather than a more specific kind of component?